packages/core/EventManager.ts

```diff
@@ -105,6 +105,14 @@ export default class EventManager {
     const isDedicated = evt.location ? isDedicatedIntegration(evt.location) : null;
 
     const results: Array<EventResult<Exclude<Event, AdditionalInformation>>> = [];
+
+    // Fallback to Cal Video if Google Meet is selected w/o a Google Cal
+    if (
+      evt.location === "integrations:google:meet" &&
```

Comment:
Might be better to use a variable for this like `DailyLocationType`
<img width="805" alt="Screenshot 2023-01-06 at 8 12 33 PM" src="https://user-images.githubusercontent.com/1780212/211034563-2b0ba357-de05-4284-998e-a988b7c4aa5a.png">